protocols/kad/src/kbucket/key.rs

```diff
@@ -182,22 +181,13 @@ impl Distance {
 mod tests {
     use super::*;
     use quickcheck::*;
-    use multihash::{wrap, Code};
-    use rand::Rng;
 
     impl Arbitrary for Key<PeerId> {
         fn arbitrary<G: Gen>(_: &mut G) -> Key<PeerId> {
             Key::from(PeerId::random())
         }
     }
 
-    impl Arbitrary for Key<Multihash> {
```

Comment:
Is there a specific reason for removing this instance?

Reply:
There is no `Key<Multihash>` instance as mentioned in the other comment.